readme.md

```diff
@@ -107,6 +107,9 @@ Change the `?username=` value to your GitHub username.
 [![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
 ```
 
+> **Note**
```

Comment:
Great addition. I would, however, make it a **Warning** admonition since it is important to know. Further, maybe we can move it under the other note. What do you think?

Reply:
Or do you think a warning is too aggressive? Then your version is okay 👍🏻.

Reply:
Yeah, i agree that **Warning** will be better there, i have updated the code. Regarding the second question, I think information about private contributions is more important than available ranks details, at least because of the large number of issues that users create on this.

Reply:
> Yeah, i agree that **Warning** will be better there, i have updated the code. Regarding the second question, I think information about private contributions is more important than available ranks details, at least because of the large number of issues that users create on this.

I agree with you on that! Thanks for the update. BTW, welcome to the team 🚀!

Reply:
> BTW, welcome to the team 🚀!

Thanks! Feel free to send me pull requests for review which you want to merge. Now you don't have to wait a while to get an approvement, probably 😄

Reply:
Nice 🚀!